### ISSUE TITLE
Build fails on armv7 w/out a clear error message

### ISSUE THREAD

**Issue by yurivict** [arbor-sim/arbor#2060]
```
[ 16% 29/173] : && /usr/bin/c++ -O2 -pipe -fstack-protector-strong -fno-strict-aliasing -O2 -pipe -fstack-protector-strong -fno-strict-aliasing   -DNDEBUG -fstack-protector-strong modcc/CMakeFiles/modcc.dir/modcc.cpp.o -o bin/modcc  lib/libmodcc.a && :
[ 17% 31/173] cd /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms && /wrkdirs/usr/ports/science/arbor/work/.build/bin/modcc -t cpu -N arb -c bbp -o /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/CaDynamics_E2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Ca_HVA.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Ca_LVAst.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Ih.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Im.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/K_Pst.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/K_Tst.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Nap_Et2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/NaTa_t.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/NaTs2_t.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/SK_E2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/SKv3_1.mod
FAILED: mechanisms/generated/bbp/bbp_catalogue.cpp mechanisms/generated/bbp/CaDynamics_E2_cpu.cpp mechanisms/generated/bbp/Ca_HVA_cpu.cpp mechanisms/generated/bbp/Ca_LVAst_cpu.cpp mechanisms/generated/bbp/Ih_cpu.cpp mechanisms/generated/bbp/Im_cpu.cpp mechanisms/generated/bbp/K_Pst_cpu.cpp mechanisms/generated/bbp/K_Tst_cpu.cpp mechanisms/generated/bbp/Nap_Et2_cpu.cpp mechanisms/generated/bbp/NaTa_t_cpu.cpp mechanisms/generated/bbp/NaTs2_t_cpu.cpp mechanisms/generated/bbp/SK_E2_cpu.cpp mechanisms/generated/bbp/SKv3_1_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/bbp_catalogue.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/CaDynamics_E2_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/Ca_HVA_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/Ca_LVAst_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/Ih_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/Im_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/K_Pst_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/K_Tst_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/Nap_Et2_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/NaTa_t_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/NaTs2_t_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/SK_E2_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp/SKv3_1_cpu.cpp 
cd /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms && /wrkdirs/usr/ports/science/arbor/work/.build/bin/modcc -t cpu -N arb -c bbp -o /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/bbp /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/CaDynamics_E2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Ca_HVA.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Ca_LVAst.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Ih.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Im.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/K_Pst.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/K_Tst.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/Nap_Et2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/NaTa_t.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/NaTs2_t.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/SK_E2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/bbp/SKv3_1.mod
[ 17% 31/173] cd /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms && /wrkdirs/usr/ports/science/arbor/work/.build/bin/modcc -t cpu -N arb -c allen -o /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/CaDynamics.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Ca_HVA.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Ca_LVA.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Ih.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Im.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Im_v2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/K_P.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/K_T.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Kd.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Kv2like.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Kv3_1.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/NaTa.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/NaTs.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/NaV.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Nap.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/SK.mod
FAILED: mechanisms/generated/allen/allen_catalogue.cpp mechanisms/generated/allen/CaDynamics_cpu.cpp mechanisms/generated/allen/Ca_HVA_cpu.cpp mechanisms/generated/allen/Ca_LVA_cpu.cpp mechanisms/generated/allen/Ih_cpu.cpp mechanisms/generated/allen/Im_cpu.cpp mechanisms/generated/allen/Im_v2_cpu.cpp mechanisms/generated/allen/K_P_cpu.cpp mechanisms/generated/allen/K_T_cpu.cpp mechanisms/generated/allen/Kd_cpu.cpp mechanisms/generated/allen/Kv2like_cpu.cpp mechanisms/generated/allen/Kv3_1_cpu.cpp mechanisms/generated/allen/NaTa_cpu.cpp mechanisms/generated/allen/NaTs_cpu.cpp mechanisms/generated/allen/NaV_cpu.cpp mechanisms/generated/allen/Nap_cpu.cpp mechanisms/generated/allen/SK_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/allen_catalogue.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/CaDynamics_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Ca_HVA_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Ca_LVA_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Ih_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Im_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Im_v2_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/K_P_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/K_T_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Kd_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Kv2like_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Kv3_1_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/NaTa_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/NaTs_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/NaV_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/Nap_cpu.cpp /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen/SK_cpu.cpp 
cd /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms && /wrkdirs/usr/ports/science/arbor/work/.build/bin/modcc -t cpu -N arb -c allen -o /wrkdirs/usr/ports/science/arbor/work/.build/mechanisms/generated/allen /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/CaDynamics.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Ca_HVA.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Ca_LVA.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Ih.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Im.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Im_v2.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/K_P.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/K_T.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Kd.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Kv2like.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Kv3_1.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/NaTa.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/NaTs.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/NaV.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/Nap.mod /wrkdirs/usr/ports/science/arbor/work/arbor-0.8/mechanisms/allen/SK.mod
ninja: build stopped: subcommand failed.
```

Version: 0.8
clang-14
FreeBSD 13.1

**Comment by thorstenhater** [comment]
Could you rerun this as
```
ninja -j1 -k1 -v
```
without deleting the progress so far?

**Comment by yurivict** [comment]
Unfortunately I don't have access to such machine. The failure was during the scheduled build on FreeBSD servers.

**Comment by thorstenhater** [comment]
Ah, ok, no matter. `ninja` runs quite some processes in parallel. Could you upload the full log file in case the message hides in some earlier output.

**Comment by thorstenhater** [comment]
I can see it, but as you noted before, there isn't really an error in there. Unfortunately, I do not have an Arm7 machine either, so I am lost here. It works on my M1 Arm chip, though. I'll ask around whether we can get access to an Arm7.

**Comment by boeschf** [comment]
could this be related to the sve backend? Maybe related to issue #2101 and the bugs fixed in PR #2096.

**Comment by thorstenhater** [comment]
I think this has been fixed in #2096
